navigator/backend/src/main/scala/com/digitalasset/navigator/console/Pretty.scala

```diff
@@ -226,23 +229,24 @@ object Pretty {
       PrettyPrimitive(constructor)
     case model.ApiList(elements) =>
       PrettyArray(
-        elements.map(e => argument(e))
+        elements.toImmArray.map(e => argument(e)).toSeq: _*
```

Comment:
Do you think we can skip an extra iteration like this? 
```suggestion
        elements.map(argument)(com.digitalasset.daml.lf.data.FrontStack.`FrontStack canBuildFrom`): _*
```
I used the fully qualified name so that this would have made sense as a self-contained single line suggestion, but please do feel free to throw this away and import the relevant class.

Reply:
Not in this case; `elements` doesn't define `map` in CanBuildFrom terms. The `toSeq` here is practically free, so at least we only have two iterations.

Reply:
> I used the fully qualified name so that this would have made sense as a self-contained single line suggestion

I would definitely only refer to these definitions via implicit resolution; that's the second intent of the spacey names, "never talk about this directly".